packages/indexer-common/src/indexer-management/actions.ts

```diff
@@ -12,10 +15,43 @@ import { Eventual, join, Logger, timer } from '@graphprotocol/common-ts'
 export class ActionManager {
   constructor(
     public allocationManager: AllocationManager,
+    public networkMonitor: NetworkMonitor,
+    private currentEpoch: number,
+    private maxAllocationEpoch: number,
     private logger: Logger,
     private models: IndexerManagementModels,
+    private allocationManagementMode?: AllocationManagementMode,
+    private autoMinBatchSize?: number,
+    private autoMaxWaitTime?: number,
```

Comment:
Looks like this value is no longer being used. (?)

Let's remove? The `affectedAllocationExpiring` check should be sufficient for forcing batch executing before the `autoMinBatchSize` is met.